string name
string args
----
string res